### ISSUE TITLE
I added proxy and got an error 

### ISSUE THREAD

**Issue by BlessedUzama** [masterking32/MasterHamsterKombatBot#278]
This is the error I got

"[Master HamsterKombat Bot][ERROR] 2024-08-30 09:39:44 [Account 1] Error: 'set' object has no attribute 'get'                           [Master HamsterKombat Bot][ERROR] 2024-08-30 09:39:44 [Account 1] Error: 'set' object has no attribute 'get'                           [Master HamsterKombat Bot][ERROR] 2024-08-30 09:39:44 [Account 1] Unable to get account basic data.".

I was using the script before without any proxy because I can't use my bank card to buy any proxy but this morning I saw a post on Twitter where an account was banned so I started looking for free proxies and i found a website that gives you 10 for signing up, and that's how i got the proxy am using but I keep getting an error and i don't know if its because the proxy is free, that's why am getting an error or am I making a mistake somewhere ?

![20240830_083427](https://github.com/user-attachments/assets/1933e4d0-281e-4408-a4d8-362c6eaef35a)

**Comment by BlessedUzama** [comment]
> #133

Thanks

**Comment by BlessedUzama** [comment]
> #133

Hey please I have a question, after I updated the script, the script keeps showing my IP address as the account IP address. Is it supposed to be like that???

**Comment by masterking32** [comment]
no, Check your proxies, After using proxies must show the proxy IP.

**Comment by DineshValor** [comment]
> This is the error I got
> 
> "[Master HamsterKombat Bot][ERROR] 2024-08-30 09:39:44 [Account 1] Error: 'set' object has no attribute 'get' [Master HamsterKombat Bot][ERROR] 2024-08-30 09:39:44 [Account 1] Error: 'set' object has no attribute 'get' [Master HamsterKombat Bot][ERROR] 2024-08-30 09:39:44 [Account 1] Unable to get account basic data.".
> 
> I was using the script before without any proxy because I can't use my bank card to buy any proxy but this morning I saw a post on Twitter where an account was banned so I started looking for free proxies and i found a website that gives you 10 for signing up, and that's how i got the proxy am using but I keep getting an error and i don't know if its because the proxy is free, that's why am getting an error or am I making a mistake somewhere ?
> 
> ![20240830_083427](https://private-user-images.githubusercontent.com/125572610/363106728-1933e4d0-281e-4408-a4d8-362c6eaef35a.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUxMjM4NjUsIm5iZiI6MTcyNTEyMzU2NSwicGF0aCI6Ii8xMjU1NzI2MTAvMzYzMTA2NzI4LTE5MzNlNGQwLTI4MWUtNDQwOC1hNGQ4LTM2MmM2ZWFlZjM1YS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODMxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgzMVQxNjU5MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kYzEyYTNjMGI5Yjg1OTFiOTMxYTllOTdhMzEwMDY1NWIxMjU1YWQ0Yjc4Nzc1MDZjZTc2NjE2OGNhYzM5YTMxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.VjFiO2neI63MYzAiLWe63kJl3wEVje_NE7IkYP-ct2E)

Where can get free proxy?

**Comment by BlessedUzama** [comment]
> > This is the error I got
> > "[Master HamsterKombat Bot][ERROR] 2024-08-30 09:39:44 [Account 1] Error: 'set' object has no attribute 'get' [Master HamsterKombat Bot][ERROR] 2024-08-30 09:39:44 [Account 1] Error: 'set' object has no attribute 'get' [Master HamsterKombat Bot][ERROR] 2024-08-30 09:39:44 [Account 1] Unable to get account basic data.".
> > I was using the script before without any proxy because I can't use my bank card to buy any proxy but this morning I saw a post on Twitter where an account was banned so I started looking for free proxies and i found a website that gives you 10 for signing up, and that's how i got the proxy am using but I keep getting an error and i don't know if its because the proxy is free, that's why am getting an error or am I making a mistake somewhere ?
> > ![20240830_083427](https://private-user-images.githubusercontent.com/125572610/363106728-1933e4d0-281e-4408-a4d8-362c6eaef35a.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUxMjM4NjUsIm5iZiI6MTcyNTEyMzU2NSwicGF0aCI6Ii8xMjU1NzI2MTAvMzYzMTA2NzI4LTE5MzNlNGQwLTI4MWUtNDQwOC1hNGQ4LTM2MmM2ZWFlZjM1YS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODMxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgzMVQxNjU5MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kYzEyYTNjMGI5Yjg1OTFiOTMxYTllOTdhMzEwMDY1NWIxMjU1YWQ0Yjc4Nzc1MDZjZTc2NjE2OGNhYzM5YTMxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.VjFiO2neI63MYzAiLWe63kJl3wEVje_NE7IkYP-ct2E)
> 
> Where can get free proxy?

https://www.webshare.io/?referral_code=iv0p6a5s3znj
